lib/trace.c

```diff
@@ -150,16 +150,25 @@ M0_INTERNAL void m0_trace_fini(void)
 
 
 /*
- * XXX x86_64 version.
+ * XXX x86_64/aarch64 version.
  */
```

Comment:
Remove this comment entirely. It's pointless now.

Reply:
Removed